solutions/standard/variables.tf

```diff
@@ -61,6 +68,122 @@ variable "secret_manager_tags" {
+  description = "The Secrets Manager instance GUID of the Secrets Manager containing your ACME private key. Required if acme_letsencrypt_private_key is not set."
+  default     = null
+}
+
+variable "private_key_secrets_manager_region" {
+  type        = string
+  description = "The region of the Secrets Manager instance containing your ACME private key. (Only needed if different from the region variable)"
+  default     = null
+}
+
+variable "acme_letsencrypt_private_key" {
+  type        = string
+  description = "The private key generated by the ACME account creation tool. Required if private_key_secrets_manager_instance_guid and private_key_secrets_manager_secret_id are not set."
+  sensitive   = true
+  default     = null
+}
+
+# Private cert engine config
+variable "private_engine_enabled" {
+  type        = bool
+  description = "Set this to true to to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance. If set to false, no iam engine will be configured for your secrets manager instance. For more details, see https://cloud.ibm.com/docs/secrets-manager?topic=secrets-manager-configure-iam-engine."
+  default     = false
+}
+
+variable "private_engine_name" {
+  type        = string
+  description = "The name of the IAM Engine used to configure an IBM Secrets Manager IAM credentials engine for an existing IBM Secrets Manager instance."
+  default     = "private-engine-sm"
+}
+
+variable "root_ca_name" {
+  type        = string
+  description = "Name of the Root CA to create for a private_cert secret engine"
+  validation {
+    condition     = length(var.root_ca_name) >= 2
+    error_message = "length of root_ca_name must be >= 2"
+  }
+  default = "root-ca"
+}
+
+variable "root_ca_common_name" {
+  type        = string
+  description = "Fully qualified domain name or host domain name for the certificate to be created"
+
+  validation {
+    condition     = length(var.root_ca_common_name) >= 4 && length(var.root_ca_common_name) <= 128
+    error_message = "length of root_ca_common_name must be >= 4 and <= 128"
+  }
+
+  validation {
+    condition     = can(regex("(.*?)", var.root_ca_common_name))
+    error_message = "root_ca_common_name must match regular expression /(.*?)/"
+  }
+  default = "terraform-modules.ibm.com"
+}
+
+variable "root_ca_max_ttl" {
+  type        = string
+  description = "Maximum TTL value for the root CA"
+  validation {
+    condition     = can(regex("^[0-9]+[s,m,h,d]{0,1}$", var.root_ca_max_ttl))
+    error_message = "root_ca_max_ttl must match regular expression /^[0-9]+[s,m,h,d]{0,1}$/"
+  }
+  default = "87600h"
+}
+
+variable "intermediate_ca_name" {
+  type        = string
+  description = "Name of the Intermediate CA to create for a private_cert secret engine"
+  default     = "intermediate-ca"
+}
+
+variable "certificate_template_name" {
+  type        = string
+  description = "Name of the Certificate Template to create for a private_cert secret engine"
```

Comment:
use proper english the the description. Don't use `private_cert`

Reply:
just came from here https://github.com/terraform-ibm-modules/terraform-ibm-secrets-manager-private-cert/blob/060eef08db1a7ffb48b7969e36cd729702da08a1/examples/default/variables.tf#L25